adapters/ion-adapter/src/main/java/com/connexta/replication/adapters/ion/IonNodeAdapter.java

```diff
@@ -119,6 +119,7 @@ public boolean createResource(CreateStorageRequest createStorageRequest) {
     builder.part("file", getResourceEntity(resource));
     builder.part("metacard", getMetadataEntity(resource.getMetadata()));
     builder.part("correlationId", resource.getId());
+    builder.part("metadata.modified", resource.getMetadata().getMetadataModified().toString());
```

Comment:
❗️The name of this part needs to exactly match the name and format specified in the IngestAPI, and there isn't a PR to add that new PR yet. This PR should not be merged until that new version of the IngestAPI is released.

Reply:
We understand, this will be done in CDR as part of this first Sprint as agreed during planning right so we can complete this story?

Reply:
Here's our API change: https://github.com/connexta/ion-ingest-api/commit/b80d72e242f1b5c9540c1ddef7a14ad43d922cc5